cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackend.java

```diff
@@ -34,37 +34,38 @@
 
     /**
      * Creates a table, given its name, if not exists in the given database.
-     * @param destination
+     * @param dataBase
      * @param tableName
```

Comment:
missing param schema?

Reply:
Fixed in [f30b89d](https://github.com/telefonicaid/fiware-cygnus/pull/2007/commits/f30b89d404aac3d99c53fd399b32d9e176fe3589)